src/main/proto/netflix/titus/agent.proto

```diff
@@ -86,9 +86,11 @@ message ContainerInfo {
 
     optional SnapshotPolicy snapshotPolicy = 10 [deprecated = true];
 
-    repeated string entrypointCmd = 11; // Deprecated. Use entrypointStr instead.
+    // deprecated: Use process instead
+    repeated string entrypointCmd = 11 [deprecated = true];
```

Comment:
Can you just remove these entirely?

Reply:
removing proto2 fields is a pain, ideally we reserve these sequence numbers to they don't get reused (at least until the removed field has been rolled out everywhere).

Big deal? I'd rather break that out in a different change.